setup.py

```diff
@@ -187,6 +187,48 @@ def cuda_version() -> Tuple[int, ...]:
     version = match.group(1).split('.')
     return tuple(int(v) for v in version)
 
+def cudnn_path() -> Tuple[str, str]:
+    """cuDNN include and library paths.
+
+    Throws FileNotFoundError if libcudnn.so is not found."""
+    cmake_module_path = str(root_path / "transformer_engine" / "cmake")
```

Comment:
https://github.com/NVIDIA/TransformerEngine/pull/831 removed our cuDNN CMake config file. We now use the one in the cuDNN C++ frontend (https://github.com/NVIDIA/cudnn-frontend/blob/main/cmake/cuDNN.cmake).